docs/nx-cloud/private/ami-setup.md

```diff
@@ -0,0 +1,143 @@
+# Setting up a dedicated NxCloud VM
+
+## AWS EC2
+
+1. Login to your AWS Console and select [the top image published here](https://us-east-1.console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;imageName=nx-cloud;owner=623002322076;sort=desc:imageName)
+2. Launch a new instance from that AMI
+3. Recommended instance type: `t3.2xlarge`
+4. You will need to SSH into the instance once it's created:
+   - Use an existing SSH key-pair that you already have installed locally.
+   - [Or create a new one and download the keys locally](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html?icmpid=docs_ec2_console#having-ec2-create-your-key-pair)
+     - Then select your new SSH pair from the list
+5. Networking:
+   - Allow the instance to receive HTTP and HTTPS traffic
+   - Allow SSH from your current IP
+6. Leave the storage options as they are (external storage of 50GB)
+7. "Launch instance"
+8. Wait 10 minutes, then navigate to your instance's IP in the browser. You should see the NxCloud dashboard!
```

Comment:
would it be worth adding in a screenshot of what the dashboard looks like?

Reply:
good shout!